examples/ecs/command_error_handling.rs

```diff
@@ -0,0 +1,51 @@
+use bevy::prelude::*;
+
+fn main() {
+    App::build()
+        .add_startup_system(handle_command_error.system())
+        .run();
+}
+
+#[derive(Debug)]
+struct AComponent(usize);
+
+fn handle_command_error(mut commands: Commands) {
```

Comment:
This system name + function is really hard to empathize with as a user. Feels *very* artificial.

Reply:
Lots of good functionality demonstrated though!

Reply:
I was mostly just trying to showcase all the functionality with error handling. 
However, yea, I guess it does seem a little artificial. 

It's difficult, because on running the example I want the error handlers to run, but this mean that the command calls have to trigger an error state.

If you have any ideas on how to improve this, it would be greatly appreciated. :)

Reply:
I'm wary of adding to much more to the example, since I feel that ultimately distracts from what the example is trying to convey: `error handling`.

Reply:
Just pushed some new changes for a different example. 
Would appreciate another look at it :)